keyboards/sixshooter/config.h

```diff
@@ -0,0 +1,131 @@
+#ifndef CONFIG_H
```

Comment:
Could you replace the top two lines with `#pragma once` and remove the endif at the end?

Reply:
Updated in 534ba4c